frontend/src/components/Workspace/Visualize/Plot/TimeSeriesPlot.tsx

```diff
@@ -167,7 +198,20 @@ const TimeSeriesPlotImple = React.memo(() => {
       width: width,
       height: height - 50,
       xaxis: {
-        range: [xrange.left, xrange.right],
+        title: {
+          text: rangeUnit,
+        },
+        titlefont: {
+          size: 12,
+          color: 'black',
+        },
+        tickfont: {
+          size: 10,
+          color: 'black',
+        },
+        range: rangeUnit === 'frames' ? [xrange.left, xrange.right] :
+          [typeof xrange.left !== 'undefined' ? xrange.left / frameRate : -2.5 ,
+            typeof xrange.right !== 'undefined' ? xrange.right / frameRate : (dataXrange.length / frameRate) + 6.8],
```

Comment:
It looks working finely.

Just to be sure, could you tell me what the values of -2.5 and 6.8 mean?
In other words, the basis for the calculations, etc.

Reply:
@ReiHashimoto these two parameters so that when converting from frames to time and vice versa, the graph will not be scaled